tests/hipscat_import/cross_match/test_macauff_runner.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+import hipscat_import.cross_match.run_macauff_import as runner
+from hipscat_import.cross_match.macauff_arguments import MacauffArguments
+
+# pylint: disable=too-many-instance-attributes
+# pylint: disable=duplicate-code
+
+def test_bad_args(dask_client):
```

Comment:
Can I sneak in one more comment? Use `@pytest.mark.dask` on tests that use a dask_client, to increase the timeout and do some other fun things.

If you run all your tests at once, you likely won't run into problems, but if you do `pytest -k macauff`, then the test will hit the default 1 second timeout just because it's creating the dask client for the first time.